jvm/src/test/java/com/microsoft/bond/protocol/BinaryStreamReaderTest.java

```diff
@@ -0,0 +1,693 @@
+package com.microsoft.bond.protocol;
+
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.EOFException;
+import java.io.IOException;
+
+import static org.junit.Assert.*;
+
+public class BinaryStreamReaderTest {
+
+    @Test
+    public void readInt8() {
+        // setup
+        byte[] inputBytes = new byte[]{
+                (byte) 0x5A,
```

Comment:
Not here.

---
In reply to: [119249657](https://github.com/Microsoft/bond/pull/465#discussion_r119249657) [](ancestors = 119249657)